hack/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+IN=origin
```

Comment:
Calling it `origin` unnecessarily puts my mind to think of openshift/origin...
How about putting the "templates" in rooted at the repo root, or give it a different name? `template/`?